stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/viewmodel/ChannelListViewModel.kt

```diff
@@ -31,8 +33,9 @@ public class ChannelListViewModel(
     private val chatDomain: ChatDomain = ChatDomain.instance(),
     private val filter: FilterObject = Filters.and(
         eq("type", "messaging"),
-        Filters.`in`("members", listOf(chatDomain.currentUser.id)),
-        Filters.or(Filters.notExists("draft"), Filters.ne("draft", true)),
+        `in`("members", listOf(chatDomain.currentUser.id)),
+        or(Filters.notExists("draft"), ne("draft", true)),
+        ne("hidden", false)
```

Comment:
Shouldn't it be the opposite? :thinking: 
Why are we filtering for "channel with hidden not equal to false"?

Reply:
This should be ne("hidden", true)).